R/get_dataset_versions.R

```diff
@@ -1,20 +1,53 @@
 #' Get data set versions
 #'
 #' @inheritParams api_url
+#' @param detail Level of detail to return. Given as a character string, it should be one of:
+#' "light" (default) or "full".
+#' Using "light" gives the following:
+#' \itemize{
+#'   \item version - version number
+#'   \item type - version type (major or minor)
+#'   \item total_rows - total number of rows in the listed version
+#'   \item date_released - release date of the version
+#'   \item version_title - name of the version release
```

Comment:
This jars a bit for some reason, would `version_release` make more sense?

Reply:
Updated